tests/t101-vector.c

```diff
@@ -1,8 +1,12 @@
 /// @file
 /// Test CeedVectorSetValue
 /// \test Test CeedVectorSetValue
+
+//TESTARGS(name="length 10") {ceed_resource} 10
```

Comment:
Can you double check the minimum number of tests we need to add a length 0 version of so we don't add more test time than we strictly need?

Reply:
sure, I figured they weren't all necessary

Reply:
okay, I think that I've slimmed it down